apps/gruenbuch/templates/gruenbuch_page.html

```diff
@@ -9,64 +9,68 @@
 {% block content %}
   <div class="container">
     <div class="row">
-      <nav class="col-sm-4 col-md-3 col-xl-2">
-        {% for chapter in page.chapters %}
-          <div class="m-1 fs-6 fw-bolder">
-            {% if chapter.url == page.active_chapter.url %}
-              <div class="chapter-link chapter-link--active">
-                <i class="fas fa-arrow-right"></i>
-                <a href="{{ chapter.url }}">
-                  {{ chapter.page_title }}
+      {% if page.isEmpty %}
+        <div>{% trans 'No chapters/content has been added, yet.' %}</div>
+      {% else %}
+        <nav class="col-sm-4 col-md-3 col-xl-2">
```

Comment:
Just becasue I don't know: Without writing col-12 here, that still happens for the smallest sizes?